packages/editor/src/components/preview-dropdown/index.js

```diff
@@ -111,7 +115,9 @@ export default function PreviewDropdown( { forceIsAutosaveable, disabled } ) {
 			label: __( 'Desktop' ),
 			icon: desktop,
 		},
-		{
+		// The "Desktop (50%)" option only works in the iframe editor,
+		// so disable it in the non-iframe editor.
+		! disableIframe && {
```

Comment:
Zoom-out should generally be disabled if there's no iframe

Reply:
Is there a way to get this value of disableIframe without passing it down in props?

Reply:
We might also need to disable the experimental zoomed-out view when the pattern category is selected and the editor is not an iframe, otherwise we'll be forcing "Desktop (50%)" even though it's not in the view options:

https://github.com/user-attachments/assets/ea7e1169-f01a-465e-9be6-5a58d86cab19

Perhaps we need to know whether the editor is an iframe or not [here](https://github.com/WordPress/gutenberg/blob/cf964194f6d6b620dc7ae6be8a23415242dd3a25/packages/block-editor/src/components/inserter/menu.js#L149-L150), is there a way?